components/dashboard/src/components/Alert.tsx

```diff
@@ -73,6 +77,12 @@ const infoMap: Record<AlertType, AlertInfo> = {
         icon: <Exclamation className="w-4 h-4"></Exclamation>,
         iconColor: "text-red-400",
     },
+    danger: {
```

Comment:
Wanted the alert w/ a darker red background for modals. Open to alternatives here.